libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Core/Binary/CDDL.hs

```diff
@@ -100,31 +100,31 @@ vrf_keyhash = "vrf_keyhash" =:= hash32
 --------------------------------------------------------------------------------
 
 hash28 :: Rule
-hash28 = "$hash28" =:= VBytes `sized` (28 :: Word64)
+hash28 = "hash28" =:= VBytes `sized` (28 :: Word64)
```

Comment:
Does the end-result `hash28 = bytes .size 2` vs `$hash28 = bytes .size 2`  make any difference at all?
I never actually understood why these types have the `$` prefix - perhaps in the cddl they weren't actually a rule (if prefixed with `$`) but just like a type alias?

Reply:
They're supposed to be "type sockets", meaning that they are meant to be extended in different places using `//=`. But really, the `$` at the start of a name is just a convention to signal intention, there's no change in semantics otherwise. 

I suppose whoever wrote the CDDL initially expected us to change the hashing format in some future era, but so far that hasn't happened. I think we don't actually want to use `/=` and `//=` anyways, since we create a new CDDL file for each era and we can just define the types from scratch every time. I'd prefer to only use the declarative subset of the CDDL spec.